doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -330,6 +330,10 @@ Libraries for networking
   * Updated the library so that it does not retry download on disconnect.
   * Fixed a race condition when starting the download.
 
+* :ref:`lib_nrf_cloud` library:
+
+  * Removed the unnecessary parameter of the :c:func:`nrf_cloud_connect` function.
```

Comment:
```suggestion
  * Removed the unused parameter of the :c:func:`nrf_cloud_connect` function.
```

I'd change unnecessary -> unused here and in commit message as unnecessary raises a question why it's unnecessary.

Reply:
I'd suggest "an unused parameter", as you do not specifically indicate which parameter was removed.